docs/istio.md

```diff
@@ -0,0 +1,201 @@
+
+## Requirements
+
+* A Kubernetes cluster 
+	+ [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/container-service) - Create a Kubernetes cluster in IBM Cloud.  The application runs in the Lite cluster, which is free of charge.  Follow the instructions [here](https://console.bluemix.net/docs/containers/container_index.html).
+	+ For local development/deployment, try a single-node cluster setup with [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+* [kubectl](https://kubernetes.io/docs/user-guide/kubectl-overview/) (Kubernetes CLI) - Follow instructions [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to install it onto your platform.
+* [helm](https://github.com/kubernetes/helm) (Kubernetes package manager) - Follow the instructions [here](https://github.com/kubernetes/helm/blob/master/docs/install.md) to install it on your platform.
+	+ If using `IBM Cloud Private`, we recommend you follow these [instructions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.0/app_center/create_helm_cli.html) to install `helm`.
+	+ If using IBM Cloud Kubernetes Service (IKS), please use the most up-to-date version of helm
+
+## Blue-Compute Istiofied
+
+For information regarding the Bluecompute architecture and further details on the Istio files, check our guide [here](https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/blob/master/docs/istio.md).
+
+## Setting up your Istio environment
+
+We will be using IBM's official [Istio Helm Chart](https://github.com/IBM/charts/tree/master/stable/ibm-istio). This chart comes with an easy way to toggle on/off different Istio components, such as Ingress and Egress gateways, with a simple boolean. Also bundled in, the chart comes with non-Istio components such as Grafana, Service Graph, and Kiali, which we can also easily toggle on/off.
+
+1) First, we must add the `IBM Cloud Charts` Helm repository with the command below:
+
+```bash
+helm repo add ibm-charts https://registry.bluemix.net/helm/ibm-charts
+```
+
+**NOTE: Before installing Istio!**  
+If using Helm version prior to **2.10.0**, either upgrade your Helm version or install IBM Istio's Custom Resource Definitions (CRDs) with the command below. You will have to wait a few seconds for the changes to be committed to the kube-apiserver:
+
+>```bash
```

Comment:
This and a lot of things below need formatting fix. Probably remove the quote blocks (>)